src/dwi/tractography/roi.cpp

```diff
@@ -118,3 +118,5 @@ Image<bool> Mask::__get_mask(const std::string &name) {
 }
 
 } // namespace MR::DWI::Tractography
+
+template MR::Transform::Transform(const MR::DWI::Tractography::Mask &);
```

Comment:
I think this one should be possible to avoid. `MR::DWI::Tractography::Mask` inherits from `Image<bool>`, which has a `transform()` member; it's just trying to pre-compute the inverse transform and share it across threads (which probably isn't worth the expense of de-referencing a shared pointer to save memory given it's just a 4x4 matrix). Having `Mask` derive from *both* `Image<bool>` *and* `MR::Transform` might make more sense?

Reply:
> Having Mask derive from both `Image<bool>` and `MR::Transform` might make more sense?

That would involve replacing all instances of `MR::Transform<MR::DWI::Tractography::Mask>::Transform(const MR::DWI::Tractography::Mask&)` with just `MR::DWI::Tractography::Mask()` right?

Reply:
I'll try to modify this myself on `dev` with a separate PR. Just won't happen this week.